CefSharp/CefSharp.csproj

```diff
@@ -76,6 +76,9 @@
     <DocumentationFile>bin\x64\Release\CefSharp.XML</DocumentationFile>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="CefSharp.Core">
```

Comment:
This will likely cause a problem, as `CefSharp.Core` has a dependency on `CefSharp`, so we're creating a circular reference here.

So we'll have to revert to using the old method of passing in the params.